dockers/docker-bgp/Dockerfile

```diff
@@ -3,11 +3,15 @@ FROM docker-base
 COPY deps/quagga_*.deb /deps/
 RUN dpkg_apt() { [ -f $1 ] && { dpkg -i $1 || apt-get -y install -f; } || return 1; } &&        \
     dpkg_apt /deps/quagga_*.deb &&                                                              \
+    apt-get update &&                                                                           \
+    apt-get -y install -f python-jinja2 python-netaddr python-ipaddr python-lxml &&             \
```

Comment:
Discussed with @lguohan @taoyl-ms, not to move to docker-base. Instead, package it and install in all the related docker images.